lib/effect/FireflyEffect.cpp

```diff
@@ -12,6 +12,15 @@ FireflyEffect::FireflyEffect() : Effect() {
 CRGB FireflyEffect::GetRGB(uint8_t led_index, uint32_t time_ms,
                            const StripDescription &strip,
                            RadioPacket *setEffectPacket) {
+  if (strip.FlagEnabled(Controller)) {
+    // For the controller, blink the lights mostly in sync
+    if (led_index % 2 == 1) {
```

Comment:
Oh yeah, this confused me. Maybe give every-other light an 1/2 cycle time offset instead?

Reply:
I think this that this makes this effect more visually distinctive on the controller - I'd like to leave as-is for now, and if it's not clear when you've got a controller in-hand, we can update it.